helm/ffc-mpdp-backend/values.yaml

```diff
@@ -43,3 +43,16 @@ readinessProbe:
   initialDelaySeconds: 10
   periodSeconds: 10
   failureThreshold: 3
+
+postgresService:
+  postgresDb: ffc_mpdp_backend
+  postgresqlHost: ffc-mpdp-backend
+  postgresPort: 5432
+  postgresSchema: public
+  postgresUser: postgres
+
+aadPodIdentity: true
```

Comment:
If this is set to true, then parent helm chart will configure service to work with a managed identity in Kubernetes